core/src/main/java/com/linecorp/armeria/common/NonBlocking.java

```diff
@@ -13,9 +13,9 @@
  * License for the specific language governing permissions and limitations
  * under the License.
  */
-package reactor.core.scheduler;
+package com.linecorp.armeria.common;
 
 /**
- * A dummy interface that makes Project Reactor recognize Armeria's event loop threads as non-blocking.
```

Comment:
Maybe we can remove this interface and just use `instanceof EventLoopThread` as a predicate?

Reply:
That was my initial thought as well. However, I considered that users might want to verify whether a thread is non-blocking in their code.
Requiring them to use `EventLoopThread` from an internal package might not be ideal. 🤔